docs/pages/tutorial/eas/manage-app-versions.mdx

```diff
@@ -19,11 +19,32 @@ An app version is composed of two values:
 
 Both Google Play Store and Apple App Store rely on developer-facing values to identify each unique build. For example, if we upload an app with the app version `1.0.0 (1)` (which is a combination of developer and user facing values), we cannot submit another build to the app stores with the same app version. Submitting builds with duplicate app version numbers results in a failed submission.
 
-## Automate app version management with EAS Build
+We can manually manage developer-facing values by adding `android.versionCode` and `ios.buildNumber` in **app.config.js**.
+
+```json app.config.js
+{
+  "ios": {
+    /* @info */
+    "buildNumber": "1"
+    /* @end */
+    /* @hide ... */ /* @end */
+  },
+  "android": {
+    /* @info */
+    "versionCode": "1"
+    /* @end */
+  }
+  /* @hide ... */ /* @end */
+}
+```
 
-> **warning** The `remote` version source as a default behavior has been introduced in `eas-cli` version `11.0.0`. If you are using an older version of the CLI, be aware that **`local` will be the default behavior**.
+After manually adding these values, we need to update them every time before creating a new production build otherwise we will run into a failed submission error. However, to eliminate the chances of running into the error, we can automate this process using EAS Build in the next section
```

Comment:
```suggestion
After manually adding these values, we need to update them every time before creating a new production build. Otherwise, we will run into a failed submission error. To eliminate the chances of running into this error, we can automate this process using EAS Build in the next section.
```